Code/Framework/AzCore/AzCore/Math/MatrixUtils.cpp

```diff
@@ -44,6 +44,22 @@ namespace AZ
         return &out;
     }
 
+    void SetPerspectiveMatrixFOV(Matrix4x4& out, float fovY, float aspectRatio)
+    {
+        float sinFov, cosFov;
+        SinCos(0.5f * fovY, sinFov, cosFov);
+        float yScale = cosFov / sinFov; //cot(fovY/2)
+        float xScale = yScale / aspectRatio;
+
+        out.SetElement(0, 0, xScale);
+        out.SetElement(1, 1, yScale);
+    }
+
+    float GetPerspectiveMatrixFOV(const Matrix4x4& m)
+    {
+        return 2.0 * atan(1.0f / m.GetElement(1, 1));
```

Comment:
nit: Can you please use `AZStd::atan` (it's in math.h in AzCore) (it aliases std::atan and as it's C++ will use overloading to work for floats so you don't need `atanf`)